docs/development/Security.md

```diff
@@ -0,0 +1,29 @@
+# Security
+We are committed to maintaining a secure and reliable platform by integrating proactive vulnerability detection and mitigation strategies into our development and release workflows. Here's  how we address security concerns at each stage of our development process.
+
+---
```

Comment:
i don't think we use these section demarcations in other markdown files but prolly fine (not sure exactly how this looks in retype)

Reply:
Yes, we don't use it anywhere. But it does look sick!